README.md

```diff
@@ -1,3 +1,6 @@
+# Comment on bounty
+Everything that requires an AWS key doesn't run. Every error related to '''collect_stats_spec''' , is because i don't have ICECAST_STATS_HOST in .env.
+Every other error is a mail error, if you run the test on your machine it should work, since it's looking for user.subscription in the database.
```

Comment:
You can just comment on the github pull request, this doesn't need to go in the README.

Reply:
Done